android/app/src/main/res/layout/fragment_item_onboarding.xml

```diff
@@ -0,0 +1,189 @@
+        app:layout_constraintBottom_toTopOf="@id/cardView"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent">
+
+        <View
+            android:id="@+id/selected_item_1_view"
+            android:layout_width="12dp"
+            android:layout_height="12dp"
+            android:layout_marginHorizontal="@dimen/margin_padding_size_xsmall"
+            android:layout_marginBottom="@dimen/margin_padding_size_small"
+            android:background="@drawable/circle_selected_item"
+            android:visibility="invisible" />
+
+        <View
+            android:id="@+id/selected_item_2_view"
+            android:layout_width="12dp"
+            android:layout_height="12dp"
+            android:layout_marginHorizontal="@dimen/margin_padding_size_xsmall"
+            android:layout_marginBottom="@dimen/margin_padding_size_small"
+            android:background="@drawable/circle_selected_item"
+            android:visibility="invisible" />
+
+        <View
+            android:id="@+id/selected_item_3_view"
+            android:layout_width="12dp"
+            android:layout_height="12dp"
+            android:layout_marginHorizontal="@dimen/margin_padding_size_xsmall"
+            android:layout_marginBottom="@dimen/margin_padding_size_small"
+            android:background="@drawable/circle_selected_item"
+            android:visibility="invisible" />
+
+    </LinearLayout>
+
+    <LinearLayout
+        android:id="@+id/unselectedItemsLinearLayout"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:orientation="horizontal"
+        app:layout_constraintBottom_toTopOf="@id/cardView"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent">
+
+        <View
+            android:id="@+id/unselected_item_1_view"
+            android:layout_width="12dp"
+            android:layout_height="12dp"
+            android:layout_marginHorizontal="@dimen/margin_padding_size_xsmall"
+            android:layout_marginBottom="@dimen/margin_padding_size_small"
+            android:background="@drawable/circle_unselected_item" />
+
+        <View
+            android:id="@+id/unselected_item_2_view"
+            android:layout_width="12dp"
+            android:layout_height="12dp"
+            android:layout_marginHorizontal="@dimen/margin_padding_size_xsmall"
+            android:layout_marginBottom="@dimen/margin_padding_size_small"
+            android:background="@drawable/circle_unselected_item" />
+
+        <View
+            android:id="@+id/unselected_item_3_view"
+            android:layout_width="12dp"
+            android:layout_height="12dp"
+            android:layout_marginHorizontal="@dimen/margin_padding_size_xsmall"
+            android:layout_marginBottom="@dimen/margin_padding_size_small"
+            android:background="@drawable/circle_unselected_item" />
+
+    </LinearLayout>
```

Comment:
Swap views with tab layout. It's better practice and helps with adding extra elements.
You can see solution here, if you want you can use some top rated libs for it
https://stackoverflow.com/questions/20586619/android-viewpager-with-bottom-dots